apis/run/v1alpha3/osimage_types.go

```diff
@@ -0,0 +1,93 @@
+// Copyright 2022 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package v1alpha3
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
+)
+
+// OSInfo describes the "OS" part of the OSImage, defined by the Operating System's name, version and CPU architecture.
+type OSInfo struct {
+	Name    string `yaml:"name"`
```

Comment:
Question: I see `type` field being mentioned in the design document. Is it intentional to not include `type` field?

Reply:
Not intentional, thanks!

Reply:
Fixed